hack/docker-builder/fedora-virt-preview.repo

```diff
@@ -0,0 +1,24 @@
+# Fedora Virt preview repository
```

Comment:
do we need any virt dependencies?

Reply:
Yes, Dockerfile fails without it:

Last metadata expiration check: 0:00:24 ago on Mon Oct 29 21:14:48 2018.
No match for argument: libvirt-devel-4.2.0
Error: Unable to find a match

Reply:
Maybe resolves after removing possible unneeded dependencies